### PR TITLE
Stricter formatting

### PR DESCRIPTION
**Pull request by ysthakur**
Ever felt like you had too much control over your life? Were you starting to doubt that you might maybe have some free will? Well, worry no more, your robot overlords are here to take care of you. This PR makes Scalafmt disregard almost all your newlines, eliminating any illusion of choice you may have had.

### CODE REVIEW

**Review by nayakrujul — APPROVED** [review]
> frick you lyxal I'll remove your commas if I damn well please

lmfao

File: .scalafmt.conf [hunk 1]
```diff
@@ -19,7 +19,16 @@ rewrite.imports.sort = original
 rewrite.imports.groups = [["scala\\.language\\..*"], ["vyxal\\..*"], ["scala\\..*", "java\\..*", "org\\.scalajs\\..*"]]
 rewrite.imports.contiguousGroups = no
 
-rewrite.trailingCommas.style = keep # frick you removing my commas.
+rewrite.trailingCommas.style = multiple # frick you removing my commas. # frick you lyxal I'll remove your commas if I damn well please
```
Comment (Lyxal) [inline .scalafmt.conf:22]: Bro really forgor he a code golfer 💀

**Review by Lyxal — CHANGES_REQUESTED** [review]
So uh what exactly do these style changes do?

Tell me and I'll probably click approve given its nothing too radical. I just want to know what it is I'd be approving.

**Comment by Lyxal** [comment]
> @Lyxal [RTFM](https://scalameta.org/scalafmt/docs/configuration.html)

No u

**Review by Lyxal — APPROVED** [review]
If you say so

**Pull request merged**